### ISSUE TITLE
405 Method Not Allowed with signup() method

### ISSUE THREAD

**Issue by lukasborawski** [pouchdb-community/pouchdb-authentication#189]
### Issue
Doing the signup call throws: 

`405 Method Not Allowed`
`error: "method_not_allowed"`
`Only DELETE,GET,HEAD,POST,PUT,COPY allowed`

The code:

```
const api = new PouchDB(`http://${process.env.apiUrl}:5984/&`, {
  skip_setup: true
})

export const createAccount = (params) => {
  const { username, password } = params
  return api.signup(username, password, (error, res) => {
    return console.log(res, error)
  })
}
```

What is wrong @pouchdb-community?

**Comment by ptitjes** [comment]
Hi @lukasborawski.

From the `process.env`, I believe you are running this on Node.js. What version ?
Also what is the server on the other side (CouchDB, Cloudant, etc) ?
What is its configuration (CORS, etc) ?
(BTW this is why we ask all those questions in the issue, to not have to guess...)

Finally, why do you put an `&` at the end of the url ?

**Comment by lukasborawski** [comment]
@ptitjes There was a problem with server CORS setup, it's ok now. 

But what with this `&` ... If i put normal path request cutting it to the `/_user/` part, no matter what I add at the end (in this case `&`) the call is working fine... 

Any hints?

**Comment by ptitjes** [comment]
Glad to here it was just a CORS setup problem!

Your URL should be `http://${process.env.apiUrl}:5984/mydatabasename`.
(There was a bug when just giving `http://${process.env.apiUrl}:5984/`, but that is now fixed in master and a version 1.0.0 is coming soon.)
